If submodules don't exist, print error message and stop build as soon as possible.

If submodules are not initialized, I get bogus error message:

```
-- Looking for pthread.h
-- Looking for pthread.h - not found
CMake Error at /home/milovidov/work/cmake-3.22.2-linux-x86_64/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Threads (missing: Threads_FOUND)                                                                                                                                                                                                                              
Call Stack (most recent call first):                                                                                                                                                                                                                                           
  /home/milovidov/work/cmake-3.22.2-linux-x86_64/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)                                                                                                                                     
  /home/milovidov/work/cmake-3.22.2-linux-x86_64/share/cmake-3.22/Modules/FindThreads.cmake:238 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)                                                                                                                                            
  cmake/linux/default_libs.cmake:35 (find_package)                                                                                                                                                                                                                             
  CMakeLists.txt:413 (include)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -62,8 +62,8 @@ set(CMAKE_DEBUG_POSTFIX "d" CACHE STRING "Generate debug library name with a pos
 # For more info see https://cmake.org/cmake/help/latest/prop_gbl/USE_FOLDERS.html
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
-# Check that submodules are present only if source was downloaded with git
-if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/.git" AND NOT EXISTS "${ClickHouse_SOURCE_DIR}/contrib/boost/boost")
+# Check that submodules are present
+if (NOT EXISTS "${ClickHouse_SOURCE_DIR}/contrib/sysroot/README.md")
     message (FATAL_ERROR "Submodules are not initialized. Run\n\tgit submodule update --init --recursive")
 endif ()
 

--- a/src/Functions/CMakeLists.txt
+++ b/src/Functions/CMakeLists.txt
@@ -48,7 +48,7 @@ endif()
 option(STRIP_DEBUG_SYMBOLS_FUNCTIONS "Do not generate debugger info for ClickHouse functions" ${STRIP_DSF_DEFAULT})
 
 if (STRIP_DEBUG_SYMBOLS_FUNCTIONS)
-    message(WARNING "Not generating debugger info for ClickHouse functions")
+    message(INFO "Not generating debugger info for ClickHouse functions")
     target_compile_options(clickhouse_functions PRIVATE "-g0")
 else()
     message(STATUS "Generating debugger info for ClickHouse functions")

